### PR TITLE
Initial support for channels on EnderFluidConduits

### PR DESCRIPTION
**Pull request by dvdmandt**
This is pretty basic support for channels on ender fluid conduits. It mainly copies item conduits for gui and rendering and does the most basic check possible to match source and target before checking target filters or target acceptance.

### CODE REVIEW

File: src/main/java/crazypants/enderio/conduit/liquid/EnderLiquidConduit.java [hunk 1]
```diff
@@ -40,6 +41,12 @@
   public static final String ICON_INSERT_KEY = "enderio:liquidConduitAdvancedOutput";
   public static final String ICON_IN_OUT_KEY = "enderio:liquidConduitAdvancedInOut";
 
+  public static final String ICON_KEY_INPUT = "enderio:itemConduitInput";
```
Comment (Glease) [inline src/main/java/crazypants/enderio/conduit/liquid/EnderLiquidConduit.java:44]: How about initialize using those constants =in ItemConduit? e.g.
```java
  public static final String ICON_KEY_INPUT = ItemConduit.ICON_KEY_INPUT;
```
or maybe just reference the constants in call site.

Comment (dvdmandt) [reply src/main/java/crazypants/enderio/conduit/liquid/EnderLiquidConduit.java:44]: The idea crossed my mind, but I figured we might want to make custom icons with some outline or something to make them slightly easier to tell apart. The idea isn't really that they are supposed to use the same icon, it just turned out to work well enough that I didn't bother making a custom version for the fluids. On the other hand, it does work alright with using the same, so I guess I could reference the existing ones.

Comment (Glease) [reply src/main/java/crazypants/enderio/conduit/liquid/EnderLiquidConduit.java:44]: I'm actually ok with any version, but I'd prefer to use my suggested version. It makes clears that this class is intentionally reusing the item conduit version of icons while making it clear this is not a typo.

**Review by Glease — APPROVED** [review]

**Comment by Glease** [comment]
Resolve merge conflict please.

**Comment by dvdmandt** [comment]
Resolved the merge conflict and fixed the references. Now we just need to fix the GUI which became too cramped with both the channel and RR button.

**Comment by dvdmandt** [comment]
I have now fixed the GUI of the fluid conduit as well, based how it's done for item conduits. It should be ready for merge.

**Pull request merged**